FrontEnd/src/components/profileList/ProfileList.jsx

```diff
@@ -0,0 +1,59 @@
+import css from "./ProfileList.module.css";
+import { List, ConfigProvider } from "antd";
+import ProfileCard from "./ProfileCard";
+
+// FIXME: change loader to the one from dev
+// FIXME: change error page to the one from dev
+
+const ListHeader = ({ number }) => (
+  <div className={css["results-header"]}>
+    <p className={css["results-header__number"]}>{number}</p>
+    <p className={css["results-header__text"]}>компаній</p>
+  </div>
+);
+
+export default function ProfileList(props) {
+  const isAuthorized = props.isAuthorized;
+  const currentPage = props.current;
+  const data = props.data;
+  const companiesFound = data.total_items;
+  const profiles = data.results;
+  const pageSize = 6;
```

Comment:
I would move it to constants and call it `PAGE_SIZE`

Reply:
done. props also destructured instead of reassigning